docs/ui/index.rst

```diff
@@ -0,0 +1,130 @@
+.. _ref_ui_overview:
+
+==
+UI
+==
+
+EdgeDB UI is a beautiful, feature-rich admin panel baked directly into all 
+EdgeDB instances.
+
+The UI automatically opens in your default browser on a ``localhost`` page
+via a single command:
+
+.. code-block:: bash
+
+  $ edgedb ui
+
+Alternatively, you can print a url to paste into the browser instead of
+automatically opening the UI:
+
+.. code-block:: bash
+
+  $ edgedb ui --print-url
+
+The UI interface for :ref:`EdgeDB Cloud <ref_cli_edgedb_cloud>` is nearly
+identical to the standard EdgeDB UI, aside from extra pages on cloud-related
+items such as org settings, billing, and usage metrics.
+
+The command to open the EdgeDB UI is a CLI command, documented
+:ref:`here <ref_cli_edgedb_ui>`.
+
+The UI is served by default by development instances. To enable the UI on a
+production instance, use the ``--admin-ui`` option with ``edgedb-server``
+or set the ``EDGEDB_SERVER_ADMIN_UI`` :ref:`environment variable
+<ref_reference_envvar_admin_ui>` to ``enabled``.
+
+The UI has three similar pages that each allow users to query the database:
+the REPL, the editor, and the data explorer. Which one to use can generally
+be decided based on your use case.
+
+The *data explorer* offers simple point-and-click access to objects without
+needing any EdgeQL, making it the recommended for:
+
+- Outright new users to EdgeDB who lack a technical background or the time
+  to familiarize themself with EdgeQL,
+- Existing users of EdgeDB looking to "walk" the database's objects without
+  needing to construct a new query each time.
```

Comment:
Maybe one more use case here is:

> - Inserting, updating, and deleting objects, since the data explorer gives you a double-confirm dialog to allow you to review the changes before you make them.

(wording completely open for editing!)

Reply:
Ah yes, good for the paranoid! I know someone who would enjoy that feature. https://github.com/edgedb/edgedb/commit/d221ea854f4cc0eb64d05533f9771745387c9727 Argh, just noticed that my WSL thinks it's two days ago. Time to restart it...